solutions/bio1/IntegerStrings/index.tsx

```diff
@@ -1,7 +1,60 @@
-import { STitle } from "components";
+import { Link, SCodeBlock, SText, STitle } from "components";
+import { brace } from "solutions/util";
 
 export const IntegerStrings = () => {
-  return <>
-    <STitle>Coming Soon!</STitle>
-  </>
-};
\ No newline at end of file
+  return (
+    <>
+      <STitle>Observation</STitle>
+      <SText>
+        This problem becomes much easier when you consider the string when $n=1$.
+      </SText>
+
+      <STitle>Sample Explanation</STitle>
+      <SText>
+        If we start at $999$, the string is $9991000100110021003...$. The $1$st digit is $9$, the $2$nd digit is also $9$, the $11$th digit is 1.
+      </SText>
+
+      <STitle>Idea</STitle>
+      <SText>
+        The digit at the $i$th position in the string when $n=1$ is the same as the digit at the $i+k$th position in the string when $n=1$, where $k$ is the number of digits before $n$ appears in the string.
```

Comment:
Please can you write `$i$-th` instead of `$i$th`? And `$(i+k)$-th` instead of `$i+k$th`.

Reply:
fixed